pkg/util/validation/exporter/exporter.go

```diff
@@ -35,7 +70,25 @@ func NewOverridesExporter(defaultLimits *Limits, tenantLimits TenantLimits) *Ove
 			[]string{"limit_name"},
 			nil,
 		),
+		logger:            log,
+		subserviceWatcher: services.NewFailureWatcher(),
+	}
+
+	if config.Ring.Enabled {
+		oeRing, err := newRing(config.Ring, log, registerer)
+		if err != nil {
+			return nil, errors.Wrap(err, "failed to create ring/lifecycler")
+		}
+		exporter.ring = oeRing
+		manager, err := services.NewManager(oeRing.lifecycler, oeRing.client)
+		if err != nil {
+			return nil, errors.Wrap(err, "failed to create service manager")
+		}
+		exporter.subserviceManager = manager
```

Comment:
[nit] Alternatively, you can do this to avoid having to assign to `exporter.ring` and `exporter.subserviceManager` separately.

```suggestion
                var err error
                
		exporter.ring, err = newRing(config.Ring, log, registerer)
		if err != nil {
			return nil, errors.Wrap(err, "failed to create ring/lifecycler")
		}

		exporter.subserviceManager, err = services.NewManager(oeRing.lifecycler, oeRing.client)
		if err != nil {
			return nil, errors.Wrap(err, "failed to create service manager")
		}
```